rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -773,6 +773,91 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
     errorLog.readAndClearErrorVector.unsafeRunSync should be(Vector.empty[InterpreterError])
   }
 
+  "let x = 1, y = 2 in { @\"result\"!(x, y) }" should "return (1, 2) to \"result\"" in {
```

Comment:
these tests should not be needed, as there should be no `let` in the ast by the time it reaches the reducer.
on the other hand, an "e2e interpreter test" executing some examples of let usage in rholang should be added.